src/main/java/org/synyx/urlaubsverwaltung/publicholiday/PublicHolidaysService.java

```diff
@@ -67,7 +67,7 @@ public DayLength getAbsenceTypeOfDate(LocalDate date, FederalState federalState)
         return List.copyOf(manager.getHolidays(from, to, federalState.getCodes()));
     }
 
-    private boolean isPublicHoliday(LocalDate date, FederalState federalState) {
+    public boolean isPublicHoliday(LocalDate date, FederalState federalState) {
```

Comment:
Das hier wieder auf private und lieber mit getAbsenceTypeOfDate arbeiten, sodass man auch halbe Feiertage darstellen kann?

Reply:
Wäre wahrscheinlich konsistenter. Allerdings weiß ich gerade nicht ob in der Ansicht überhaupt auch halbe Urlaubstage angezeigt werden.

Reply:
ja, habs mal umgebaut. Commit kommt